_drafts/2017-05-16-lingtypology.Rmd

```diff
@@ -0,0 +1,199 @@
+---
+name: lingtypology
+layout: post_discourse
+title: Easy linguistic mapping with lingtypology
+date: 2017-05-16
+authors:
+  - name: George Moroz
+    url: https://www.hse.ru/en/org/persons/103489498#teaching
+tags:
+- R
+- linguistic mapping
+- clld
+- glottolog
+output: 
+  html_document: 
+    keep_md: yes
+---
+
+As all other types of visualization, linguistic mapping has two main goals: data presentation and data analysis. The most common purpose for which linguistic maps are used, is simply pointing to the location of one or more languages of interest (presentation). A more sophisticated task is showing the distribution of particular linguistic features or their combination among languages of a certain area (presentation and analysis). There are three linguistic subdisciplines that use maps for visualization: linguistic typology, areal linguistics and dialectology. 
+
```

Comment:
Can you be more explicit in second para by saying that `lingtypology` does presentation and analysis (or just presentation if I'm wrong about that)